plugins/catalog-backend/src/ingestion/processors/github/github.ts

```diff
@@ -330,7 +332,7 @@ export async function queryWithPaging<
   GraphqlType,
   OutputType,
   Variables extends {},
-  Response = QueryResponse,
+  Response = QueryResponse
```

Comment:
Having the `,` might be a convention they follow, I'd keep it if there isn't a good reason to remove it

Reply:
Good catch, corrected.